Library/Formula/ola.rb

```diff
@@ -2,8 +2,8 @@
 
 class Ola < Formula
   homepage "http://www.openlighting.org/ola/"
-  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.3/ola-0.9.3.tar.gz"
-  sha1 "f6a81087761218063a4bb8006b73ffa407cd0170"
+  url "https://github.com/OpenLightingProject/ola/releases/download/0.9.5/ola-0.9.5.tar.gz"
+  sha1 "e9b0d8e4e3e4acd696cd7a0877d72d593727a7bc"
```

Comment:
You can either do ` brew fetch /path/to/formula.rb -fs ` which will output the ` sha1 ` and ` sha256 ` or you can download it manually and then ` openssl dgst -sha256 /path/to/file.tar.gz `.

> So, in addition to changing the value, I should change the key "sha1" to be "sha256", I expect?

Yup! Thanks :heart: